pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ['setuptools ~= 41.4', 'setuptools_scm ~= 3.3', 'wheel ~= 0.33.6']
+build-backend = 'setuptools.build_meta'
```

Comment:
https://github.com/pypa/setuptools/blob/master/setuptools/build_meta.py#L105 ensures this wont break/supports old metadata

this note is for myself